osu.Game/Database/RealmAccess.cs

```diff
@@ -595,8 +595,8 @@ public IDisposable BlockAllOperations()
                 {
                     if (updateRealm == null)
                     {
-                        // null realm means the update thread has not yet retrieved its instance.
-                        // we don't need to worry about reviving the update instance in this case, so don't bother with the SynchronizationContext.
+                        // null context means the update thread has not yet retrieved its context.
+                        // we don't need to worry about reviving the update context in this case, so don't bother with the SynchronizationContext.
```

Comment:
guessing that this here is an inadvertent partial revert of f30894840c709505464cc2b903f4a8870feb7fd4

Reply:
Fixed this in the merge commit.